flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaSinkITCase.java

```diff
@@ -428,13 +434,13 @@ private void createTestTopic(String topic, int numPartitions, short replicationF
                 admin.createTopics(
                         Collections.singletonList(
                                 new NewTopic(topic, numPartitions, replicationFactor)));
-        result.all().get();
+        result.all().get(1, TimeUnit.MINUTES);
```

Comment:
why do we need this change?

Reply:
Don't remember exactly why I added tha
will remove it for now